ua.Relocator.lua

```diff
@@ -1450,7 +1450,7 @@ function modifier(subs,sel,act)
 		end
 		if text:match("\\i?clip") and res.rnd=="all" or text:match("\\i?clip") and res.rnd=="clip" then
 		  for klip in text:gmatch("\\i?clip%([^%)]+%)") do
-		    if klip:match("m") then rrr=0 else rrr=rr end
+		    rrr=rr
```

Comment:
'rrr' should just be replaced with 'rr' in the gsub call, no need to reassign it. 
```suggestion

```